src/app/routes/index.js

```diff
@@ -6,6 +6,11 @@ const routes = [
     exact: true,
     component: Article,
   },
+  {
+    path: '/article/amp',
```

Comment:
This needs to be `/article/amp/:id` so that you can visit the various articles on the AMP route.

Reply:
Ok. I thought it was a bit odd there wasn't an `:id` path variable.